pkg/kn/commands/source/duck/multisourcelist_test.go

```diff
@@ -83,8 +83,9 @@ func newSourceUnstructuredObjWithSink(name, apiVersion, kind string) *unstructur
 			"spec": map[string]interface{}{
 				"sink": map[string]interface{}{
 					"ref": map[string]interface{}{
-						"kind": "Service",
-						"name": "foo",
```

Comment:
Only kind and name are not enough here. An error will appear: "Unable to get the Subscriber's URI"